content/docs/02-foundations/04-tools.mdx

```diff
@@ -103,6 +103,7 @@ There are several providers that offer pre-built tools as **toolkits** that you
 - **[Composio](https://docs.composio.dev/javascript/vercel)** - Composio provides 250+ tools like GitHub, Gmail, Salesforce and [more](https://composio.dev/tools).
 - **[Interlify](https://www.interlify.com/docs/integrate-with-vercel-ai)** - Convert APIs into tools so that AI can connect to your backend in minutes.
 - **[Freestyle](https://docs.freestyle.sh/integrations/vercel)** — Tool for your AI to execute JavaScript or TypeScript with arbitrary node modules.
+- \*\*[JigsawStack](http://www.jigsawstack.com/docs/integration/vercel) - JigsawStack provides over 30+ small custom fine tuned models available for specific uses.
```

Comment:
oh i see what is wrong. you need `**` at the beginning (not escaped) but **also** after the link (see other items)